wbooks/src/app/style.ts

```diff
@@ -0,0 +1,7 @@
+import { StyleSheet } from 'react-native';
+
```

Comment:
It's ok but add a TODO to move this constants inside `AppNavigator` folder in the TabBar PR